src/main.rs

```diff
@@ -23,6 +23,8 @@ const fn default_port() -> u16 {
     3000
 }
 
+const MAX_DESCRIPTION_LENGTH: usize = 640;
```

Comment:
We can change this if you want, I just wanted the channel to still be easily scrollable.

Reply:
this is fine, the number i chose was based on measuring messages from the official github<>discord integration, but we can use anything we like.